sdk/communication/Azure.Communication.NetworkTraversal/CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0 (2022-02-07)
```

Comment:
This release has breaking changes and goes directly to GA without publishing a beta package first.
Was this the consensus with the Azure SDK board? (To me, the changes seem small, so not super opposed)
@tg-msft @KrzysztofCwalina

Reply:
Yes, we always prefer no breaking changes between the last beta and GA.  I agree it's fine in this case though.  It's just an API simplification rather than a bunch of new APIs, the actual churn  is very small, the use cases are all for more advanced customers, and based on the daily download count I doubt you'd get much customer feedback with another beta.